tools/genpki/genpki.go

```diff
@@ -0,0 +1,223 @@
+package main
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"time"
+
+	"github.com/ThalesIgnite/crypto11"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	var (
+		rootPriv, interPriv crypto.PrivateKey
+		rootPub, interPub   crypto.PublicKey
+		slots               []uint
+		noHSM               bool
+		err                 error
+	)
+	flag.BoolVar(&noHSM, "no-hsm", false,
+		"generate keys locally instead of using an hsm")
+	flag.Parse()
+
+	rootKeyName := []byte(fmt.Sprintf("csroot%d", time.Now().Unix()))
+	if !noHSM {
+		p11Ctx, err := crypto11.Configure(&crypto11.PKCS11Config{
+			Path:       "/usr/lib/softhsm/libsofthsm2.so",
+			TokenLabel: "test",
+			Pin:        "0000",
```

Comment:
worth mentioning these values in the readme

Reply:
fixed